cli/pack/opts.go

```diff
@@ -15,9 +21,45 @@ const (
 	Docker = "docker"
 )
 
+func initAppsInfo(cliOpts *config.CliOpts, cmdCtx *cmdcontext.CmdCtx, packCtx *PackCtx) error {
```

Comment:
It would be nice to add a comment. Up to you

Reply:
Comment is added.